docs/configuration.rst

```diff
@@ -66,6 +66,13 @@ The following environment variables for the tracer are supported:
      description: |
          Enables sending :ref:`telemetry <Instrumentation Telemetry>` events to the agent.
 
+   DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED:
+     type: Boolean
+     default: False
+     description: |
+         Historically datadog trace ids were 64 bit integers. This configuration enables the generation of 128 bit trace ids to better align
```

Comment:
```suggestion
         This configuration enables the generation of 128 bit trace ids to better align
```

we don't need the context of the configuration option, just what it does and any limitations (like lack of b3/w3c propagation support)